mainapp/models.py

```diff
@@ -3,61 +3,58 @@
 from django.core.validators import MinLengthValidator, int_list_validator
 from datetime import datetime
 
+
 # Create your models here.
 
 class Comment(models.Model):
-    id = models.BigAutoField('id',primary_key=True)
+    id = models.BigAutoField('id', primary_key=True)
     content = models.TextField('content', max_length=400, blank=False)
     created_at = models.DateTimeField('created_at', default=datetime.utcnow)
     deleted_at = models.DateTimeField('deleted_at', null=True)
-    user_id = models.ForeignKey('User_agent', on_delete=models.SET_NULL, null=True)
+    user_id = models.OneToOneField(settings.AUTH_USER_MODEL, on_delete=models.SET_NULL, null=True)
     product_id = models.ForeignKey('Product', on_delete=models.SET_NULL, null=True)
 
     def __str__(self):
         return self.content
 
+
 class Product(models.Model):
     id = models.BigAutoField('id', primary_key=True)
     title = models.CharField('title', max_length=100, blank=False)
     content = models.TextField('content', max_length=400, blank=False)
     created_at = models.DateTimeField('created_at', default=datetime.utcnow)
     deleted_at = models.DateTimeField('deleted_at', null=True)
-    is_active = models.BooleanField('is_active',default=False)
-    autor_id = models.ForeignKey('User_agent', on_delete=models.SET_NULL, null=True)
+    is_active = models.BooleanField('is_active', default=False)
+    autor = models.OneToOneField(settings.AUTH_USER_MODEL, on_delete=models.SET_NULL, null=True)
```

Comment:
1. `author` (typo).
2. `models.ForeignKey` (many-to-one). Same as for `Comment` (see above).